apps/frontend/src/app/pages/company/gpts/gpts.component.html

```diff
@@ -0,0 +1,347 @@
+<div class="container mx-auto min-h-screen px-3 sm:px-0">
+  <div class="pb-4 pt-6 flex flex-wrap justify-between gap-6">
+    <h2 class="font-sans font-medium text-4xl text-base-100">GPTs</h2>
+    @if(gptType === 'FINETUNING'){
+    <button
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/7b1f56a9-86e4-48db-a62e-07231ada3c41)

The reactive form for these inputs is missing

Reply:
I forgot to mention that the select input will need it aswell